### ISSUE TITLE
Visualization: DSL to scale the Image

### ISSUE THREAD

**Issue by darcy-shen** [eto-ai/rikai#523]
``` python
## Scale the image
image = Image("path/to/xyz.jpg")

image1 = image * 2
image2 = image * 0.5
image3 = image * (1, 2)
image4 = image * (2, 1)
image5 = image * (1, 0.5)
image6 = image * (0.5, 1)

## Scale the image and then append the RelativeBox2d
(image * (x, y)) | RelativeBox2d(0.a, 0.b)
```

+ https://learnopencv.com/image-resizing-with-opencv/
+ https://pytorch.org/vision/stable/auto_examples/plot_transforms.html#resize

**Comment by changhiskhan** [comment]
Hmmm does this mean we add a Transform abstraction to Rikai.viz? Then you can scale/translate/add noise etc ?

**Comment by eddyxu** [comment]
I can see how resize work.  How does general transform work , in what context? 

`image * (1, 2)` deserves a PR for sure.

**Comment by darcy-shen** [comment]
closed by #694
